### PR TITLE
Chapter3/homework/1

### PR DESCRIPTION
**Pull request by littleNoodle**
use property and List<T>

### CODE REVIEW

File: Src/BootCamp.Chapter/Items/Chestpiece.cs [hunk 1]
```diff
@@ -6,11 +6,34 @@ public class Chestpiece
         private decimal _price;
         private float _weight;
 
-        public Chestpiece(string name, decimal price, float weight)
+        public Chestpiece(string name, decimal price, float weight, float defense)
```
Comment (norarrsgd) [inline Src/BootCamp.Chapter/Items/Chestpiece.cs:9]: I think this class can be simplified further. Technically your approach is still correct. But I don't see any valid reason to keep it this way. You can try

```csharp
public class Chestpiece
{
    public string Name { get; set; }
    public decimal Price { get; set; }
    public float Weigh { get; set; }
    public float Defense { get; set; }
}
```

Then in your calling method/code you can instantiate it like so
```csharp
Chestpiece chestPiece = new Chestpiece
{
    Name = "Some Name",
    Price = 10.5,
    Weight = 4.5,
    Defense = 1.2
};
```

I know I only commented on this file, but I think this applies to most of your classes ✌🏻

LMK if you have further questions

**Review by norarrsgd — CHANGES_REQUESTED** [review]
✌🏻

File: Src/BootCamp.Chapter/Items/Chestpiece.cs (hunk 1 above)
Comment (littleNoodle) [reply Src/BootCamp.Chapter/Items/Chestpiece.cs:9]: Thanks for your kind advice, I will modify the constructor function, O(∩_∩)O~

**Review by Almantask — APPROVED** [review]
Looks good!

**Pull request closed without merging**